server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.static(`${__dirname}/dist`));
+
+app.listen(PORT, function () {
```

Comment:
Для колбеков обычно используют стрелочные функции, они получаются гораздо минималистичнее)

```suggestion
app.listen(PORT, () => {
```